examples/port-2-google-connection/variables.tf

```diff
@@ -1,10 +1,12 @@
 variable "equinix_client_id" {
   description = "Equinix client ID (consumer key), obtained after registering app in the developer platform"
   type        = string
+  sensitive = true
 }
 variable "equinix_client_secret" {
   description = "Equinix client secret ID (consumer secret), obtained after registering app in the developer platform"
   type        = string
+  sensitive = true
 }
 #Google Provider
 variable "google_region" {
```

Comment:
Missing `sensitive = true` on goodle credentials.

Reply:
We don't have any Google credentials for passing secrets directly. Instead, we are passing secrets using file names.